README.md

```diff
@@ -107,13 +107,34 @@ explore a commercial market, please contact us for a license agreement.
 
 
 
-## Development
+# Development
 For code formatting, [`black`](https://github.com/psf/black) and [`isort`](https://github.com/PyCQA/isort) are used.
 Type hints should be added as much as possible.
 Types are checked using [`mypy`](https://github.com/python/mypy).
 
 Before code is pushed, make sure that the `make lint` command succeeds, which runs `black`, `isort`, `flake8` and `mypy`.
 
+## Branches and releases
```

Comment:
Maybe mention that these steps are just for maintainers? Since the repo is public.

Reply:
Good point, I updated the text accordingly. Also, I realized it's probably a good idea to restrict pushing tags (triggering PyPI publishing) to maintainers only. Any idea how we could do this?

Reply:
Actually I think only maintainers are able to push tags anyway, so it's fine as it is. Am I correct?

Reply:
Yes that's correct, we can leave it as is.